requirements_tests.txt

```diff
@@ -6,3 +6,4 @@ pytest
 pytest-cov
 requests_mock
 tox
+pathlib
```

Comment:
This is not necessary in modern Python, you can drop this.

Reply:
This also true for requirements.txt?

Reply:
I am talking about this one line. Not the whole file. That one is still necessary.

Reply:
Long day, I'll get that reverted here shortly.